include/TimeLineWidget.h

```diff
@@ -117,41 +109,7 @@ class TimeLineWidget : public QWidget, public JournallingObject
 		return m_autoScroll;
 	}
 
-	BehaviourAtStopState behaviourAtStop() const
-	{
-		return m_behaviourAtStop;
-	}
-
-	void setBehaviourAtStop (int state)
-	{
-		emit loadBehaviourAtStop (state);
-	}
-
-	bool loopPointsEnabled() const
-	{
-		return m_loopPoints == LoopPointState::Enabled;
-	}
-
-	inline const TimePos & loopBegin() const
-	{
-		return ( m_loopPos[0] < m_loopPos[1] ) ?
-						m_loopPos[0] : m_loopPos[1];
-	}
-
-	inline const TimePos & loopEnd() const
-	{
-		return ( m_loopPos[0] > m_loopPos[1] ) ?
-						m_loopPos[0] : m_loopPos[1];
-	}
-
-	inline void savePos( const TimePos & pos )
-	{
-		m_savedPos = pos;
-	}
-	inline const TimePos & savedPos() const
-	{
-		return m_savedPos;
-	}
+	auto timeline() const -> Timeline& { return *m_timeline; }
```

Comment:
I think it's fine - the timeline and the view are separate, and modifying the former doesn't modify the latter. However, looking back on this, I think it would be better if I removed it, and replaced uses of it with access via `Song::getTimeline` instead.